rfc/rfc-65/rfc-65.md

```diff
@@ -0,0 +1,248 @@
+## Proposers
+
+- @stream2000
+- @hujincalrin
+- @huberylee
+- @YuweiXiao
+
+## Approvers
+
+## Status
+
+JIRA: [HUDI-5823](https://issues.apache.org/jira/browse/HUDI-5823)
+
+## Abstract
+
+In some classic hudi use cases, users partition hudi data by time and are only interested in data from a recent period
+of time. The outdated data is useless and costly, we need a lifecycle management mechanism to prevent the
+dataset from growing infinitely.
```

Comment:
Guess you are talking about GDPR?

Reply:
To clarify: this is very different, the proposal for this doc only yields target partitons, it does not remove the whole partition.